src/requestTracing/constants.ts

```diff
@@ -43,6 +43,7 @@ export enum RequestType {
     STARTUP = "Startup",
     WATCH = "Watch"
 }
+export const FAILOVER_REQUEST_TYPE = "FailoverRequest";

```

Comment:
The tag is wrong. Please check .net provider.

https://github.com/Azure/AppConfiguration-DotnetProvider/blob/preview/src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/RequestTracingConstants.cs#L35